apps/extension/src/shared/idriss/components/send/form.tsx

```diff
@@ -39,23 +51,66 @@ export const Form = ({
         <Controller
           control={formMethods.control}
           name="amount"
-          render={({ field, fieldState }) => {
+          render={({ field }) => {
             return (
-              <IdrissMoneyInput
-                value={field.value}
-                onChange={field.onChange}
-                errorMessage={fieldState.error?.message}
+              <DesignSystemForm.Field
+                {...field}
+                value={field.value.toString()}
+                onChange={(value) => {
+                  field.onChange(Number(value));
+                }}
+                label="Amount ($)"
+                numeric
               />
             );
           }}
         />
+
+        <div className="mt-2 grid grid-cols-3 gap-4">
```

Comment:
wdyt about using creating a constant like `predefinedAmounts = [1,2,5]` and using `predefinedAmounts.map((amount => ....`?

Reply:
I added TODO to tackle this later